content/docs/command-reference/run.md

```diff
@@ -10,8 +10,6 @@ usage: dvc run [-h] [-q | -v] [-d <path>] [-o <path>] [-O <path>]
                [-p <params>] [-m <path>] [-M <path>] [-f <filename>]
                [-w <path>] [--no-exec] [-y] [--overwrite-dvcfile]
                [--ignore-build-cache] [--remove-outs] [--no-commit]
-               [--outs-persist OUTS_PERSIST]
-               [--outs-persist-no-cache OUTS_PERSIST_NO_CACHE]
```

Comment:
This 2 options appeared after merging the params PR. These are old and no longer existent right? @shcheklein @efiop because they're not documented in the cmd ref. Just wondering what's going on here...

Reply:
Hmmm, I see they come out in `dvc run -h` so are these new? Can someone give me their description or link to the original PR so I reinstate them here and add them to the cmd ref?

Reply:
I've reinstated them and added basic descriptions to the cmd ref in afeba4e9 for now.

Reply:
We were deliberately not documenting this :) But it's fine, I think we can keep that simple explanation.

Reply:
Kk